### DISCUSSION TITLE
Fundamentos do Blazor com .Net 8 - Injeção de Dependência

### DISCUSSION THREAD

**Discussion by manuGalo** [balta-io/forum#458] [Tira Dúvidas] [answered]
Olá, 
Na aula de Injeção de Dependência ao tentar inspecionar o banco vi que o app.db está com formato de documento não compátivel, ou o documento contém caracteres binários, abrindo um aplicativo diferente. Estou utilizando o Visual Studio 2022 e tenho o System.Data.SQLite instalado no projeto.
<img width="452" alt="Captura de tela 2024-01-08 071550" src="https://github.com/balta-io/forum/assets/75847547/6bf68e66-aa97-4d6c-9298-dec03e5d4b1c">

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @manuGalo, como vai?
Acredito que o Visual Studio nativamente não tem suporte aos arquivos do SQLite.

Você pode usar este cara aqui:
https://sqlitebrowser.org/
